src/anomalib/utils/benchmarking/utils.py

```diff
@@ -3,16 +3,29 @@
 # Copyright (C) 2022 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 
-
+import logging
 import random
 import string
 from glob import glob
 from pathlib import Path
 
 import pandas as pd
-import wandb
-from comet_ml import Experiment
-from torch.utils.tensorboard.writer import SummaryWriter
+
+logger = logging.getLogger(__name__)
+
+
+try:
+    import wandb
+except ImportError:
+    logger.warn("Could not find wandb. To log to wandb, ensure that you have wandb installed.")
```

Comment:
would it be an idea to have a util function like `try_import(package: str)` that does this?